vibe_code_apps_20/examples/workspace_tools/compare_open_meteo_bills.py

```
@@ -0,0 +1,104 @@
+    obs_t.append(bt)
+    sk.append(ek)
+    st.append(et)
+
+elec_g14 = nmbe_cvrmse(obs_k, sk)
+gas_g14 = nmbe_cvrmse(obs_t, st)
+ann_e, ann_g = sum(sk), sum(st)
+ann_be, ann_bg = sum(obs_k), sum(obs_t)
+eui = (ann_e * KWH_TO_KBTU + ann_g * THERM_TO_KBTU) / AREA
+beui = (ann_be * KWH_TO_KBTU + ann_bg * THERM_TO_KBTU) / AREA
+out = {
+    "weather": "Open-Meteo AMY (weather_observed → amy.epw), hours Dec2024–Nov2025",
```

Ruff (0.15.21)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)